gemfiles/Gemfile.common.rb

```diff
@@ -39,8 +39,6 @@
   end
 
   if RUBY_VERSION >= '2.3'
```

Comment:
No need for this condition if we don't support Ruby versions earlier than this.

I assume [these](https://github.com/quipper/mongomapper/pull/1/files#diff-354f30a63fb0907d4ad57269548329e3R9) are the only versions supported now?

Reply:
Right, I think we can remove this. :+1: